reversed.go

```diff
@@ -0,0 +1,19 @@
+package do
+
+import "reflect"
+
+/*
+Reversed returns the values of a given interface on a reversed basis.
+*/
+func Reversed(iterable interface{}) interface{} {
+	iterValue := getIterable(iterable)
+	returnValue := reflect.MakeSlice(iterValue.Type(), iterValue.Len(), iterValue.Len())
+
+	j := 0
```

Comment:
you can get rid of this by doing the following during assignment
(outside the loop)
iterLen := iterValue.Len()

(inside the loop)
returnValue.Index(iterLen - i).Set(iterValue.Index(i))